Automattic/wp-calypso
# Remove eventbrite connection details from marketing connections

The api used by the eventbrite plugin has been decommissioned from 20 Feb 2020, so the marketing connection for it can't be used now.

#### Changes proposed in this Pull Request

* Removes the eventbrite related code from the marketing/connections section of the client.

#### Testing instructions

* Check out this PR and build and run the client locally
* Go to the /marketing/connections page for one of your sites and make sure the Eventbrite option does not display
* Double check that the other connections are all still functioning correctly

Before:

<img width="741" alt="Screen Shot 2020-02-25 at 2 27 39 PM" src="https://user-images.githubusercontent.com/3629020/75206318-04e82b00-57db-11ea-8769-96b3b0815b65.png">

After:

<img width="744" alt="Screen Shot 2020-02-25 at 2 20 30 PM" src="https://user-images.githubusercontent.com/3629020/75205972-2399f200-57da-11ea-9fbc-aa206da22f16.png">
